### ISSUE TITLE
McStas 2019 Survey Feedback 3

### ISSUE THREAD

**Issue by willend** [McStasMcXtrace/McCode#728]
Multiple scattering in powder sample :-)

**Comment by willend** [comment]
True, this is built-in when using Union. It is on the other hand also on my list of possible improvements of the stand-alone PowderN that it could get this feature (in non-concentric arrangements only I guess).

**Comment by farhi** [comment]
The powder mode in Single_crystal may also be a solution.

**Comment by willend** [comment]
Some of this was in fact added (along the lines of the MS “model” from Incoherent) - also, use NCrystal.
